stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -208,14 +208,16 @@ swift_reflection_addReflectionInfo(SwiftReflectionContextRef ContextRef,
     std::cerr << "reserved field in swift_reflection_info_t is not zero\n";
     abort();
   }
-  
+
   ReflectionInfo ContextInfo{
     sectionFromInfo<FieldDescriptorIterator>(Info, Info.field),
     sectionFromInfo<AssociatedTypeIterator>(Info, Info.associated_types),
     sectionFromInfo<BuiltinTypeDescriptorIterator>(Info, Info.builtin_types),
     sectionFromInfo<CaptureDescriptorIterator>(Info, Info.capture),
     sectionFromInfo<const void *>(Info, Info.type_references),
-    sectionFromInfo<const void *>(Info, Info.reflection_strings)};
+    sectionFromInfo<const void *>(Info, Info.reflection_strings),
+    // TODO: Conformance section
```

Comment:
`addReflectionInfo` is deprecated, so this can be left as `nullptr, 0` without the TODO. Clients should use `addImage` instead, which lets us decide which sections we need to locate.